packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -189,7 +190,12 @@ export const prepareVaultManagerKit = (
 
   const { priceAuthority, timerService } = zcf.getTerms();
 
+  let singletonMade = false;
+
   const initState = () => {
+    !singletonMade || Fail`vaultManager singleton can be made just once`;
+    singletonMade = true;
```

Comment:
Why do we need this protection? What mistake could cause `initState` to be executed twice?

Reply:
`makeVaultManager` being called again. It's not a likely mistake but I figured if it did happen you'd want an error.

I saw it as only upside, but I suppose the downside is code complexity and memory cost of the boolean. The boolean is minor, especially for singletons, but something. For the code complexity, it's best to be consistent across singletons. I found `initEmpty` you suggested, which I suppose is a clear enough signal.